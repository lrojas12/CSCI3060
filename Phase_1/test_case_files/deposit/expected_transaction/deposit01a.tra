10 Tarzan               00001          S 
04 Tarzan               00001 00020.00   
00 Tarzan               00001          S 