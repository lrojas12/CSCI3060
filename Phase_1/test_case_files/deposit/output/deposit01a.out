Welcome to Watermelon Banking System
Please login to begin or type in “help” for more information
Mode in which you wish to login as:
You are currently logged in as an administrator
Enter account holder’s name:
Enter account number:
Enter amount to deposit:
Funds have been successfully added to the account acc_number
New balance: new_balance
You have successfully logged out of the administrator account 