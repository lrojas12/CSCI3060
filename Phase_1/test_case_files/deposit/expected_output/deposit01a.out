Welcome to Watermelon Banking System
Please login to begin or type in “help” for more information.
Mode in which you wish to login as:
Enter account holder’s name:
You are currently logged in as Tarzan
Bank Account Number: 00001
Balance: $1000.00
Transaction Payment Plan: non-student
Status: active
Enter account number:
Enter amount to deposit:
Funds have been successfully added to your account.
New balance: $1020.00
You have successfully logged out of your account.